clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/deploy/description/EnableDisableDescriptionTrait.groovy

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.deploy.description
+
+trait EnableDisableDescriptionTrait {
+  String serverGroupName
+
+  Integer getDesiredPercentage() {
+    throw new IllegalArgumentException("The selected provider hasn't implemented enabling/disabling by percentage yet");
```

Comment:
I picked `IllegalArgument` since it would be surfaced as an `IllegalArgument` to the operation, enable/disable which is supported.